File: testprojects/python_main/pipelines/PYTHON_SG_SRC/code/com/sg_src/main/graph/all_type_main_pythonsg/Subgraph_4_1_1/very_complex_dataset.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from .config import *
from com.sg_src.main.udfs.UDFs import *

def very_complex_dataset(spark: SparkSession) -> DataFrame:
    return spark.read.format("parquet").load("dbfs:/Prophecy/qa_data/parquet/verycomplexdata.parquet")

File: testprojects/scala_main/pipelines/SCALA_BASIC/code/src/main/scala/com/scala/main/job1/graph/very_complex_dataset.scala
package com.scala.main.job1.graph

import io.prophecy.libs._
import com.scala.main.job1.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object very_complex_dataset {

  def apply(context: Context): DataFrame =
    context.spark.read
      .format("parquet")
      .load("dbfs:/Prophecy/qa_data/parquet/verycomplexdata.parquet")

}
